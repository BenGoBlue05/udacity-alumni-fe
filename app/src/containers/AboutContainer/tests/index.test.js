import About from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';


const data = [
  {
    label: 'Anything you want',
    image: 'https://upload.wikimedia.org/wikipedia/commons/3/33/White_square_with_question_mark.png',
    heading: 'Hei You!',
    githubUrl: 'https://github.com/udacityalumni',
    description: 'There surely is something you can do to make this better! : )',
  },
];

describe('<About />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <About contributorVoices={data} />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
