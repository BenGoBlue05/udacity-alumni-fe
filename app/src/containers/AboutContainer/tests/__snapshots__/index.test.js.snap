exports[`<About /> should render with default props 1`] = `
<Box
  align="center"
  announce={false}
  direction="column"
  focusable={true}
  pad="large"
  responsive={true}
  tag="div">
  <Split
    fixed="false"
    flex="both"
    priority="left"
    showOnResponsive="priority">
    <Article
      align="center"
      direction="column"
      pad="large" />
    <Sidebar
      direction="column"
      full={true}>
      <Box
        align="center"
        announce={false}
        direction="column"
        focusable={true}
        pad="large"
        responsive={true}
        tag="div">
        <Heading
          strong={true}
          tag="h2">
          Contributor Voices
        </Heading>
        <CardComponent
          description="There surely is something you can do to make this better! : )"
          githubUrl="https://github.com/udacityalumni"
          heading="Hei You!"
          image="https://upload.wikimedia.org/wikipedia/commons/3/33/White_square_with_question_mark.png"
          label="Anything you want" />
      </Box>
    </Sidebar>
  </Split>
</Box>
`;
